core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/Parameter.kt

```diff
@@ -19,7 +19,7 @@ object ParameterManager {
 
 sealed class Parameter<T : ValidType>(
     private val value: Any,
-    private val parameterName: String? = null,
+    private val parameterName: String?,
```

Comment:
The class is sealed and can therefore not be created. I thought it might be better if the default value would be in the sub-classes, which can be created.